akka-docs/src/test/java/jdocs/stream/IntegrationDocTest.java

```diff
@@ -309,6 +310,41 @@ public Receive createReceive() {
   }
   //#ask-actor
 
+  //#actorRefWithAck-actor
+  static class Ack {}
```

Comment:
Do we have a convention to wrap those in a `Messages` enclosing class or something similar? The `static` looks kind of weird now.

Reply:
Ouch sorry I have not addressed this feedback! Should I follow up with it you think?